lib/services/elasticsearch.js

```diff
@@ -934,13 +934,11 @@ class ElasticSearch extends Service {
           errors: []
         };
 
-        let
-          i = 0,
-          row;
+        let idx = 0
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)